libraries/lib-wave-track/WaveClip.h

```diff
@@ -283,6 +283,19 @@ class WAVE_TRACK_API WaveClip final :
       size_t iChannel, sampleCount start, size_t length,
       bool mayThrow = true) const override;
 
+   /*!
+    * @brief Request interval samples within [t0, t1). `t0` and `t1` are
+    * truncated to the clip's play start and end. Stretching influences the
+    * number of samples fitting into [t0, t1), i.e., twice as few for twice as
```

Comment:
Here and elsewhere, "half as many" would read better than "twice as few"

Reply:
sure